docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/enforcement.adoc

```diff
@@ -99,160 +105,119 @@ For every failed scan result you can view the latest Pull Request (PR) of your r
+Optionally, you can access Enforcement in *Application Security* by selecting *Settings* > *Application Security* under 'Configure' in the left menu > *Enforcement Rules*.
 +
 image::application-security/enfor-2.png[]
 
+
 [.task]
 
 [#modify-default-enforcement]
-=== Modify Default Enforcement
+=== Modify Default Enforcement Configurations
 
-While you have the option to customize the default enforcement configuration, it's important to note that any modifications made will apply universally to all repositories on the console.
+Default enforcement configurations can be modified for any code category. These changes will be applies universally to all integrated repositories.
 
-NOTE: The default enforcement configurations cannot be deleted
+NOTE: Default enforcement configurations cannot be deleted.
 
 [.procedure]
 
-. <<access-enforcement,Access>> default enforcement configuration.
+// . <<access-enforcement,Access>> default enforcement configuration.
 
-. Modify the default configuration.
+. <<access-enforcement,Access Enforcement>> settings.
 
-.. Select a code category.
+. In a code category, drag the threshold rule arrow (Hard fail, Soft fail, Comments bot) to your required severity.
 +
-In this example, see Vulnerabilities.
+In this example for the 'Vulnerabilities' code category, a Hard fail has been downgraded from critical to high vulnerability detection. 
 +
-image::application-security/enfor-3.png[]
+image::application-security/enforcement-modify.png[]
 
-.. Select the severity threshold corresponding to the code category.
-+
-In this example, severity is *High* for Vulnerabilities.
-+
-image::application-security/enfor-4.png[]
-+
-You can choose to continue modifying other code categories or conclude with a single modification.
-+
-You can also choose to <<turn-off-run-rule-scan-for-a-code-category,turn off>> the severity threshold of a code category.
+. Optional: add a label. see <<add-label,Add Labels>> below for more information about labels. 
+
+. Select *Save*.
+
+[.task]
+
+[#add-label]
+=== Add Labels
+
+Labeling enforcement rules provides a filtering mechanism. When you apply a label to enforcement rules, those rules will only apply to repositories with the corresponding label and will not affect other repositories. Labels can be reused and assigned across multiple enforcement rules, allowing for the grouping of policies and the creation of enforcement rules for these labeled groups. You can apply labels to both default and custom policies when you create the policy or by editing an existing policy.
 
-.. Select *Save* the modified enforcement configuration.
+[.procedure]
+
+. <<access-enforcement,Access Enforcement>> settings.
+. Click on the *Label* field to select an available label from the drop-down menu > *Save*.
++ 
+NOTES:
```

Comment:
These "NOTES" should not be part of the procedure. Move the content to another paragraph with the a heading such as Tips maybe. The heading should also not be in all caps.

Reply:
Done